Source/Cogworks.Umbraco.Essentials/Controllers/Actions/ResponsiveImageSurfaceController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Web.Mvc;
+using Cogworks.Essentials.Constants;
+using Cogworks.Essentials.Extensions;
+using Cogworks.Umbraco.Essentials.Builders.Interfaces;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Web.Mvc;
+
+namespace Cogworks.Umbraco.Essentials.Controllers.Actions
+{
+    public class ResponsiveImageSurfaceController : SurfaceController
+    {
+        private readonly IResponsiveImageBuilder _responsiveImageBuilder;
+
+        public ResponsiveImageSurfaceController(IResponsiveImageBuilder responsiveImageBuilder) => _responsiveImageBuilder = responsiveImageBuilder;
+
+        [ChildActionOnly]
+        public virtual ActionResult ResponsiveImage(IPublishedContent image, string cropPrefix, string altText = null, IReadOnlyDictionary<string, string> breakpoints = null,
```

Comment:
Move `IReadOnlyDictionary<string, string> breakpoints = null` to new line

Reply:
I would place each parameter in new line.

Reply:
Hmm in most case its ok, but if we have lots of parameters then it starts to be unreadable. Maybe we need to define our standard to keep f.e. max 4 in one line (if they name are not to long).